clusterloader2/pkg/config/template_provider.go

```diff
@@ -21,12 +21,15 @@ import (
 	"bytes"
 	"fmt"
 	"io/ioutil"
+	"os"
 	"path/filepath"
+	"reflect"
 	"regexp"
 	"strings"
 	"sync"
 	"text/template"
 
+	goErrors "github.com/go-errors/errors"
```

Comment:
convention is to use small letters only for aliases

Reply:
Done. 

In the ideal world we'd have a lint check for such things. 
@oxddr, you were looking at some golint alternatives, right? Would it be possible to configure them to detect such things?

Reply:
Enforcing naming convention could be done using the current linter. However we have many violations in our codebase, so I've put the change on hold. I will have a look at it again in spare time.